ASWF/CLA-individual.md

```diff
@@ -0,0 +1,33 @@
+Individual Contributor License Agreement ("Agreement")
+
+Thank you for your interest in the Open Shading Language Project a Series of
+LF Projects, LLC (hereinafter "Project"). In order to clarify the
+intellectual property licenses granted with Contributions from any corporate
+entity to the Project, LF Projects, LLC ("LF Projects") is required to have
+an Individual Contributor License Agreement (ICLA) on file that has been
+signed by each contributing individual. (For legal entities, please use the
+Corporate Contributor License Agreement (CCLA).)
```

Comment:
Same as above - first reference `Open Shading Language Project` and second `Open Shading Language Project a Series of LF Projects, LLC`

Reply:
OK, thanks, I will make those edits.
I thought I copied these straight from other projects, so you may want to check openexr and ocio. I could have botched it, or those may also be the same.

Reply:
I pushed an update implementing these instructions.

Reply:
Ok - you're gonna kill me - but let's revert back to what you originally provided to make this consistent with other projects. Sorry :(

Reply:
hahahaha
no problem